Doc/library/dbm.rst

```diff
@@ -58,41 +58,33 @@ the Oracle Berkeley DB.
 
 .. function:: open(file, flag='r', mode=0o666)
 
-   Open the database file *file* and return a corresponding object.
+   Open a database and return the corresponding database object.
 
-   If the database file already exists, the :func:`whichdb` function is used to
-   determine its type and the appropriate module is used; if it does not exist,
-   the first module listed above that can be imported is used.
+   :param file:
+      The database file to open.
 
-   The optional *flag* argument can be:
-
-   .. csv-table::
-      :header: "Value", "Meaning"
+      If the database file already exists, the :func:`whichdb` function is used to
+      determine its type and the appropriate module is used; if it does not exist,
+      the first submodule listed above that can be imported is used.
+   :type file: :term:`path-like object`
 
-      ``'r'`` (default), |flag_r|
-      ``'w'``, |flag_w|
-      ``'c'``, |flag_c|
-      ``'n'``, |flag_n|
+   :param str flag:
+      * ``'r'`` (default), |flag_r|
+      * ``'w'``, |flag_w|
+      * ``'c'``, |flag_c|
+      * ``'n'``, |flag_n|
 
-   The optional *mode* argument is the Unix mode of the file, used only when the
-   database has to be created.  It defaults to octal ``0o666`` (and will be
-   modified by the prevailing umask).
+   :param int mode:
+      The Unix file access mode of the file (default: octal ``0o666``),
+      used only when the database has to be created.
 
    .. versionchanged:: 3.11
       *file* accepts a :term:`path-like object`.
 
-
-The object returned by :func:`open` supports the same basic functionality as a
+The object returned by :func:`~dbm.open` supports the same basic functionality as a
```

Comment:
Explicitly use ´~dbm.open´, so there is no doubt we refer to dbm.open, and not the builtin.